docs/articles_en/documentation/openvino-ir-format/operation-sets/operation-specs/sort/search-sorted-15.rst

```diff
@@ -0,0 +1,72 @@
+SearchSorted
+===============
+
+
+.. meta::
+  :description: Learn about SearchSorted - a sorting and maximization
+                operation, which requires two input tensors.
+
+
+**Versioned name**: *SearchSorted-15*
+
+**Category**: *Sorting and maximization*
+
+**Short description**: Determines the indices in the innermost dimension of a sorted sequence where elements should be inserted to maintain order.
+
+**Detailed description**: *SearchSorted* operation determines the indices in the innermost dimension of a sorted sequence where elements should be inserted to maintain order. The operation is based on the binary search algorithm. The operation is performed on two input tensors: the first tensor contains a monotonically increasing sequence on the innermost dimension, and the second tensor contains the search values. The operation returns a tensor with the same shape as the second input tensor, containing the indices.
+
+**Attributes**
+
+* *right*
+* **Description**: If False, set the first suitable index. If True, return the last suitable index for given value. Default is False.
+  * **Range of values**: true or false
+  * **Type**: boolean
+  * **Default value**: false
+  * **Required**: *no*
+
+**Inputs**:
+
+* **1**: ND input tensor of type *T*, containing monotonically increasing sequence on the innermost dimension **Required.**
+
+* **2**: ND input tensor of type *T*, containing the search values. **Required.**
```

Comment:
Scalar is also a special case of nd tensor, isn't it?

Reply:
In fact scalar can be interpreted as 0D case of ND. In my opinion the description for the second input is ok, but for the same reason, it should be mentioned that the first input (sorted sequence) can be ND but not scalar (0D). Also we need to add (maybe as a note above) that if sorted sequence is 1D, then the values can have any shape, otherwise the rank should be equal.

Reply:
Why 1st input cannot be a scalar then? It works with pytorch.

Reply:
> Why 1st input cannot be a scalar then? It works with pytorch.

Based on my tests it doesn't. 
Cases like in0: scalar, in1: scalar or in0: scalar, in1: 1D are leading to exception that it's not allowed.
And preventing 0D sequence input is reasonable for me, as index of insertion to "scalar" shape could be misleading.
Could you share your example?

Reply:
(some discussion has happened via external channels) Fixed.

Reply:
> Also we need to add (maybe as a note above) that if sorted sequence is 1D, then the values can have any shape, otherwise the rank should be equal.

Ok, what about this note?

Reply:
Fixed - hope it is fine now, otherwise please push a commit with whatever changes you also want to avoid any confusion please.